src/components/Input/index.js

```diff
@@ -171,23 +171,23 @@ const Input = ({ fieldData, name, value, fieldHidden, subfield, fromNameField, .
                     size
                 )}
                 id={name}
-                maxLength={type === 'phone' ? 26 : type === 'text' ? 256 : maxLength || 524288} // 524288 = 512kb, avoids invalid prop type error if maxLength is undefined.
+                maxLength={maxLength ? maxLength : type === 'phone' ? 26 : type === 'text' ? 256 : 524288} // 524288 = 512kb, avoids invalid prop type error if maxLength is undefined.
                 name={name}
                 placeholder={placeholder}
                 {...register(name, {
                     value: defaultValue,
                     required: !fieldHidden ? isRequired && strings.errors.required : false,
-                    maxLength: type === 'phone' ? {
+                    maxLength: maxLength > 0 && maxLength ? {
+                        value: maxLength > 0 && maxLength,
+                        message:
+                            maxLength > 0 &&
```

Comment:
Same here: maxLength is greater than 0 if this line of code runs.

Reply:
updated, thanks!!